indico/modules/events/registration/api.py

```diff
@@ -57,6 +57,7 @@ def _process_PATCH(self):
 
         if 'checked_in' in request.json:
             self._registration.checked_in = bool(request.json['checked_in'])
+            signals.event.registration_checkin_updated(self._registration)
```

Comment:
Did you test this? I don't think signals implement `__call__` so you need to call `.send()` on them, e.g.

    signals.event.registration_checkin_updated.send(self._registration)